Apps/WebClient/src/ClientApp/src/components/timeline/note.vue

```diff
@@ -326,8 +329,17 @@ export default class NoteTimelineComponent extends Vue {
     private deleteNote(): void {
         this.text = this.entry.id;
         if (confirm("Are you sure you want to delete this note?")) {
-            this.noteService.deleteNote(this.entry.toModel());
-            this.onNoteDeleted(this.entry);
+            this.noteService
+                .deleteNote(this.entry.toModel())
+                .then(() => {
+                    this.onNoteDeleted(this.entry);
+                })
+                .catch((err) => {
+                    this.errorDescription = ErrorTranslator.getHumanReadable(
+                        err
+                    );
+                    this.hasErrors = true;
```

Comment:
Can "hasErrors" and "errorDescription" be encapsulated in a generic front-end/typescript class as I assume we will use it everywhere?

Reply:
yes, there is a task about moving the error display to vuex